.eslintrc.js

```diff
@@ -1,5 +1,29 @@
 module.exports = {
+  root: true,
+  env: {
+    node: true
+  },
   extends: [
-    '@nuxtjs/eslint-config-typescript'
```

Comment:
well that ends up with different code style and rules for main project and this one, not good